app/views/prison/feedbacks/new.html.erb

```diff
@@ -0,0 +1,32 @@
+<% content_for :header do %>
+  <%= t('.title') %>
+<% end %>
+
+<% if current_user %>
+  <%= render 'prison/dashboards/banner' %>
+  <%= render 'prison/dashboards/navigation' %>
+<% end %>
+
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <p><%= t('.intro') %></p>
+  </div>
+</div>
+
+<%= form_for @feedback, url: prison_feedbacks_path do |f| %>
+
+  <fieldset>
+    <%= f.hidden_field :referrer %>
+
+    <%= single_field f, :body, :text_area, cols: 50, rows: 5, class: 'form-control' %>
+
+    <%= single_field f, :email_address, :email_field, class: 'form-control' %>
```

Comment:
The login page is now a separate service so it can't be there. I think you mean the staff guide. Must say I'd be more comfortable long term putting this behind the login, but we can discuss.